src/consentManager.ts

```diff
@@ -168,3 +168,23 @@ export const BackendSyncOption = makeEnum({
 /** Override type */
 export type BackendSyncOption =
   typeof BackendSyncOption[keyof typeof BackendSyncOption];
+
+/**
+ * Whether or not to run cross-domain sync
+ *
+ * default: 'on'
+ */
+export const LocalSyncOption = makeEnum({
+  /** use private consent sync only */
```

Comment:
```suggestion
  /** use private sync only */
```
Sync syncs more than just consent